test/environment_test.rb

```diff
@@ -32,4 +34,8 @@ def test_uses_environment_variables_in_production_environment
     assert_equal 1234, backend.port
     assert_equal :datadog, backend.implementation
   end
+
+  def test_uses_env_when_rails_does_not_respond_to_env
+    assert_equal ENV['ENV'], StatsD::Instrument::Environment.environment
+  end
```

Comment:
Can you also add a test that ensures everything works as expected if `Rails.env` is available?